libs/build_mapl.sh

```diff
@@ -61,13 +63,12 @@ CMAKE_OPTS=${STACK_mapl_cmake_opts:-""}
 
 cmake .. \
       -DCMAKE_INSTALL_PREFIX=$prefix \
-      -DCMAKE_MODULE_PATH="${CMAKEMODULES_ROOT}/Modules;${ESMA_CMAKE_ROOT}" \
+      -DCMAKE_MODULE_PATH="${ESMA_CMAKE_ROOT};${CMAKEMODULES_ROOT}/Modules;${ECBUILD_ROOT}/share/ecbuild/cmake" \
```

Comment:
Builds fine on Hera

Reply:
I don't think it is a good idea to send the esma cmake and ecbuild from here. I am sure it is explicitly included in mapl.

Reply:
`ESMA_CMAKE_ROOT` is required to find ESMA_cmake. The `esma.cmake` script (included in `MAPL/CmakeLists.txt`) uses `find_package(ecbuild)` to load ecBuild.

Reply:
ecbuild is handled internally, but if you set it here then it doesn't get set again when ecbuild is loaded and override FindNetCDF.